system-parachains/people/people-kusama/src/identity_ops.rs

```diff
@@ -0,0 +1,182 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#[frame_support::pallet]
+pub mod pallet_identity_ops {
+	use crate::*;
+	use frame_support::pallet_prelude::*;
+	use frame_system::pallet_prelude::*;
+	use pallet_identity::Judgement;
+
+	type IdentityPallet = pallet_identity::Pallet<Runtime>;
+
+	#[pallet::pallet]
+	pub struct Pallet<T>(PhantomData<T>);
+
+	#[pallet::config]
+	pub trait Config: frame_system::Config {}
+
+	#[pallet::error]
+	pub enum Error<T> {
+		/// No judgement to clear.
+		NotFound,
+	}
+
+	pub(crate) type Identity = (
+		pallet_identity::Registration<
+			<Runtime as pallet_balances::Config>::Balance,
+			<Runtime as pallet_identity::Config>::MaxRegistrars,
+			<Runtime as pallet_identity::Config>::IdentityInformation,
+		>,
+		Option<BoundedVec<u8, <Runtime as pallet_identity::Config>::MaxUsernameLength>>,
+	);
+
+	/// Alias for `IdentityOf` from `pallet_identity`.
+	#[frame_support::storage_alias(pallet_name)]
+	pub(crate) type IdentityOf =
+		StorageMap<IdentityPallet, Twox64Concat, AccountId, Identity, OptionQuery>;
+
+	#[pallet::call]
+	impl<T: Config> Pallet<T> {
+		/// Clear requested judgements that do not have a corresponding deposit reserved.
+		///
+		/// This is successful only if the `target` account has judgements to clear. The transaction
+		/// fee is refunded to the caller if successful.
+		#[pallet::call_index(0)]
+		#[pallet::weight(Pallet::<T>::weight_clear_judgement())]
+		pub fn clear_judgement(
+			_origin: OriginFor<T>,
+			target: AccountId,
+		) -> DispatchResultWithPostInfo {
+			let identity = IdentityPallet::identity(&target).ok_or(Error::<T>::NotFound)?;
+			let (removed, identity) = Self::do_clear_judgement(&target, identity);
+			ensure!(removed > 0, Error::<T>::NotFound);
+
+			IdentityOf::insert(&target, identity);
```

Comment:
First thought was that this was inserting in the local pallet storage. Never seen the first generic in `StorageMap` be other than `_` before (like [here](https://github.com/polkadot-fellows/runtimes/blob/4d6808aba652b9d2343f1135cef0ff2c5d810483/system-parachains/people/people-kusama/src/identity_ops.rs#L49)).

Maybe @ggwpez / @gupnik can check for my sanity that this will update the right storage value?

Reply:
Yeah, we use a similar patter in migrations as well.